src/boot/efi/pe.h

```diff
@@ -3,17 +3,26 @@
 
 #include "efi.h"
 
-EFI_STATUS pe_memory_locate_sections(
-                const void *base,
-                const char * const sections[],
-                size_t *addrs,
-                size_t *sizes);
+EFI_STATUS pe_kernel_info(const void *base, uint32_t *ret_compat_address);
+
+struct PeSectionDescriptor {
```

Comment:
for our own structs we usually (but not always) typedef things to their own name, in particular if we name things in CamelCase.